### PR TITLE
Add brianmcarey to reviewers in OWNERS

### PR DESCRIPTION
**Pull request by brianmcarey**
/cc @dhiller @rmohr 

Signed-off-by: Brian Carey <bcarey@redhat.com>

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -5,17 +5,16 @@ filters:
...
       - rmohr
       - tiraboschi
       - vladikr
+      - brianmcarey
     approvers:
       - aglitke
       - davidvossel
       - dhiller
       - enp0s3
-      - fgimenez
```
Comment (rmohr) [inline OWNERS:18]: @brianmcarey would you mind moving Federico to the `emeritus_approvers` section?

**Review by rmohr — CHANGES_REQUESTED** [review]
One small request.

File: OWNERS (hunk 1 above)
Comment (brianmcarey) [reply OWNERS:18]: yes I will make that change

Comment (rmohr) [reply OWNERS:18]: Thanks! :+1:

**Pull request merged**